services/comm/db/migrate/20200110055523_add_recipient_id_to_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddRecipientIdToMessages < ActiveRecord::Migration[6.0]
+  def change
+    add_column :messages, :recipient_id, :integer
```

Comment:
for the time being i'd avoid adding the recipient id to the database and I'd just make it a virtual attribute.

Reply:
If we assign `recipient_id` to a virtual attribute, we won't be able to know if the message is set to send to recipient id or phone number cause eventually it only records the phone number value.

What do you think?